packages/proto/transaction/transaction.proto

```diff
@@ -3,8 +3,9 @@ syntax = "proto3";
 option go_package = "../transaction";
 import "google/protobuf/timestamp.proto";
 
+
 message Transaction{
-  string id =1;
+  string id =1 ;
```

Comment:
it's better to add uuid validation like this:
 string account_id = 1 [(validate.rules).string.uuid = true];

Reply:
yes I agree, however, I will add it later on with bazel :)